### PR TITLE
Fixes #15492 - Truncate scap content title

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

File: app/views/scap_contents/_list.html.erb [hunk 1]
```diff
@@ -1,18 +1,14 @@
-<table class="table table-bordered table-striped table-two-pane">
+<table class="table table-bordered table-striped table-two-pane  table-fixed">
```
Comment (ares) [inline app/views/scap_contents/_list.html.erb]: nit: double space

File: app/views/scap_contents/_list.html.erb [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
   <% for content in @contents %>
     <tr>
-      <td>
-        <%= content.title %>
-      </td>
-      <td>
-        <%= content.original_filename %>
-      </td>
+      <td class="ellipsis"><%= content.title %></td>
+      <td  class="ellipsis"><%= content.original_filename %></td>
```
Comment (ares) [inline app/views/scap_contents/_list.html.erb]: nit: double space

**Comment by ares** [comment]
works perfectly, just two nits because of my OCD, then good to merge

**Comment by shlomizadok** [comment]
It is good that your OCD helps my ADHD :) Fixed.

**Comment by ares** [comment]
That's the right diversity :-) Thanks @shlomizadok, merging.

**Pull request merged**